diff -ur davfs2-1.4.0.orig/src/cache.c davfs2-1.4.0/src/cache.c
--- davfs2-1.4.0.orig/src/cache.c	2009-05-01 09:28:16.000000000 +0200
+++ davfs2-1.4.0/src/cache.c	2009-05-30 23:22:21.000000000 +0200
@@ -2502,7 +2502,7 @@
     if (fd <= 0) {
         syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_ERR),
                _("can't create cache file %s"), node->cache_path);
-        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_ERR), strerror(errno));
+        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_ERR), "%s", strerror(errno));
         free(node->cache_path);
         node->cache_path = NULL;
         return EIO;
@@ -2536,7 +2536,7 @@
     if (fd <= 0) {
         syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_ERR),
                _("can't create cache file %s"), dir->cache_path);
-        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_ERR), strerror(errno));
+        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_ERR), "%s", strerror(errno));
         free(dir->cache_path);
         dir->cache_path = NULL;
         return EIO;
Only in davfs2-1.4.0/src: cache.c.orig
diff -ur davfs2-1.4.0.orig/src/mount_davfs.c davfs2-1.4.0/src/mount_davfs.c
--- davfs2-1.4.0.orig/src/mount_davfs.c	2009-04-29 20:36:30.000000000 +0200
+++ davfs2-1.4.0/src/mount_davfs.c	2009-05-30 23:22:21.000000000 +0200
@@ -1783,7 +1783,7 @@
     char *cmdline;
     if (argz_create(argv, &cmdline, &len) == 0) {
         argz_stringify(cmdline, len, ' ');
-        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_DEBUG), cmdline);
+        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_DEBUG), "%s", cmdline);
         free(cmdline);
     }
 }
Only in davfs2-1.4.0/src: mount_davfs.c.orig
diff -ur davfs2-1.4.0.orig/src/webdav.c davfs2-1.4.0/src/webdav.c
--- davfs2-1.4.0.orig/src/webdav.c	2009-05-01 19:00:25.000000000 +0200
+++ davfs2-1.4.0/src/webdav.c	2009-05-30 23:22:21.000000000 +0200
@@ -1405,7 +1405,7 @@
             written++;
         }
         *cpos = '\0';
-        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_DEBUG), (char *) cookie);
+        syslog(LOG_MAKEPRI(LOG_DAEMON, LOG_DEBUG), "%s", (char *) cookie);
     }
     return written;
 }
Only in davfs2-1.4.0/src: webdav.c.orig
